pkgs/by-name/uv/uv/package.nix

```diff
@@ -70,14 +67,19 @@ python3Packages.buildPythonApplication rec {
     "uv"
   ];
 
+  doInstallCheck = true;
+  installCheckPhase = ''
```

Comment:
Maybe using `testers.testVersion` would be better (and thus optional)?

Reply:
Instead of `version` you mean ?

Reply:
```nix
passthru.tests.version = testers.testVersion { package = uv; };
```

That should be enough

Reply:
> Maybe using `testers.testVersion` would be better (and thus optional)?

I did not know this, but indeed this is a perfect fit.